.pre-commit-config.yaml

```diff
@@ -41,9 +41,17 @@ repos:
   - id: doc8
 
 - repo: https://github.com/pycqa/isort
-  rev: 5.10.1
+  rev: 5.12.0
   hooks:
   - id: isort
+    args: ['--order-by-type', '--length-sort', "--profile", "black", "--filter-files"]
```

Comment:
I understand it was important to use higher version of isort in order to make code running on gdds.

Are below changes also needed? If they are not needed please remove.

Reply:
Verified as not needed - removed